samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -966,22 +972,110 @@ For your device to successfully connect to nRF Cloud, you must `onboard it <nRF
 
 To install the repository, clone or download it and `install the required packages <nRF Cloud Utilities Prerequisites>`_.
 
+.. _nrf_cloud_multi_service_provisioning_service:
+
+Provisioning with the nRF Cloud Provisioning Service
+====================================================
+
+The nRF Cloud Provisioning Service is only compatible with nRF91x1 devices.
+For devices using the nRF9160, proceed to :ref:`nrf_cloud_multi_service_standard_onboarding`.
+
+If you have :ref:`enabled support <nrf_cloud_multi_service_building_provisioning_service>` for the `provisioning service <nRF Cloud Provisioning Service_>`_, you can provision and onboard your device in one of two ways:
+
+1. Using auto-onboarding, the easiest method.
+
+   The nRF Cloud Provisioning Service auto-onboarding is currently compatible with CoAP and REST but not MQTT connectivity to nRF Cloud; for that, use scripted onboarding.
+
+   This method is done with a serial terminal program and the nRF Cloud website.
+   The device ID used in nRF Cloud must use the UUID format and not the 'nrf-\ *IMEI*\ ' format.
+   See `device claiming <nRF Cloud device claiming_>`_ for more information.
+
+#. Using scripted onboarding, described as follows.
+
+   First, :ref:`create a self-signed CA certificate <nrf_cloud_multi_service_create_self_signed_ca>`.
+
+   Then, complete the following steps for each device you wish to onboard:
+
+   1. Make sure your device is plugged in, and that this sample has been flashed to it.
+   #. Use the :file:`claim_and_provision_device.py` Python script :ref:`you installed <nrf_cloud_multi_service_install_nrf_utils>` to provision and onboard your device
+
+      .. tabs::
+
+         .. group-tab:: MQTT
+
+               .. parsed-literal::
+                  :class: highlight
+
+                  python3 claim_and_provision_device.py --api_key "\ *your_api_key*\ " --ca="self\_\ *self_cert_serial*\ \_ca.pem" --ca_key="self\_\ *self_cert_serial*\ \_prv.pem" --install_ca --unclaim --id_imei --id_str "nrf-"
+
+               Where the :file:`.pem` files are the self-signed CA certificate and private key files :ref:`you created <nrf_cloud_multi_service_create_device_cred_locally>` and *your_api_key* is your `nRF Cloud REST API key`_.
+
+         .. group-tab:: CoAP
+
+               .. parsed-literal::
+                  :class: highlight
+
+                  python3 claim_and_provision_device.py --api_key "\ *your_api_key*\ " --ca="self\_\ *self_cert_serial*\ \_ca.pem" --ca_key="self\_\ *self_cert_serial*\ \_prv.pem" --install_ca --unclaim --id_imei --id_str "nrf-" --coap
+
+               Where the :file:`.pem` files are the self-signed CA certificate and private key files :ref:`you created <nrf_cloud_multi_service_create_device_cred_locally>` and *your_api_key* is your `nRF Cloud REST API key`_.
+               The ``--coap`` option indicates that the device needs the CoAP root CA installed.
+               See below for more details.
+
+      .. note::
```

Comment:
If the contents starting here is supposed to be included in the CoAP tab, indent as the text above.

Reply:
No, it applies to both tabs, thanks.